spark/src/main/scala/org/apache/spark/sql/delta/hooks/UpdateCatalog.scala

```diff
@@ -261,11 +264,11 @@ object UpdateCatalog {
   // This is the encoding of the database for the Hive MetaStore
   private val latin1 = Charset.forName("ISO-8859-1")
 
-  // Maximum number of characters that a catalog can store.
-  val MAX_CATALOG_TYPE_DDL_LENGTH = 4000
   val ERROR_KEY = "delta.catalogUpdateError"
   val LONG_SCHEMA_ERROR: String = "The schema contains a very long nested field and cannot be " +
```

Comment:
When this error is thrown, make sure that it says which configuration to update so that users can self-service this.

Reply:
@tdas We don't throw an error when this truncation happens, we just log the error message in an HMS table property. This is because this sync to the catalog is a best-effort async operation. I could try to create a user-visible log here.

Reply:
@tdas There are a few ways in which we can surface this. I will address this specific issue in a follow up PR.